### ISSUE TITLE
Fab subtitler cannot be run with NVDA

### ISSUE THREAD

**Issue by nvdaes** [nvaccess/nvda#11331]
### Steps to reproduce:
1. Try to run Fab Subtitler using NVDA.

### Actual behavior:
A dialog shows the following error:

> Privileged instruction.
After accepting, a second dialog shows:
> "Access violation at address 76050EB8 in module 'user32.dll'. Read of address 18085B8D."]

### Expected behavior:
Fab Subtitler should be run without issues, like using Narrator.

#### NVDA installed/portable/running from source:
Installer  or portable.

#### NVDA version:
NVDA 2020.2beta2

#### Windows version:
Tested on Windows 7 and 10.

#### Name and version of other software in use when reproducing the issue:
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes: NVDA 2020.1, with same behavior.

#### If addons are disabled, is your problem still occuring?
Yes.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes.
[nvda.log](https://github.com/nvaccess/nvda/files/4869608/nvda.log)

**Comment by nvdaes** [comment]
Fab is a proprietary product found at
https://www.fab-online.com/subtitling/bcastxcd/
I use it in my job as an employer of a company dedicated to accessibility, in case this info is useful for further support or testing.
I will send this to Twitter mentioning them, since I'm testing with their products and for job.
Thanks

**Comment by Adriani90** [comment]
cc: @feerrenrut

**Comment by Adriani90** [comment]
@nvdaes are you still having this issue with NVDA 2024.3 Beta?

**Comment by nvdaes** [comment]
@Adriani90 , at this moment I'm not using Fab, so I cannot check this.

**Comment by seanbudd** [comment]
Closing as stale, cannot proceed without more information
